client/Fireport/Assets/Scripts/Game/GamePieceFactory.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Game {
+    [CreateAssetMenu(menuName = "ScriptableObjects/GamePieceFactory")]
+    public class GamePieceFactory : ScriptableObject {
```

Comment:
The GamePieceFactory ScriptableObject will need to be configured to contain references to all of the different game pieces in the game.  We could also create multiple factories to do things like skinning (Halloween-theme factory vs Christmas-theme factory vs Default factory).